### PR TITLE
Version 1.1.0

### PR DESCRIPTION
**Pull request by ajmyyra**
Changelog upgrade and some readme fixes.

### CODE REVIEW

**Comment by kaminek** [comment]
you forgot to update the links, https://github.com/UpCloudLtd/upcloud-cli/pull/90/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR61

**Comment by ajmyyra** [comment]
@kaminek: which links? our readme just has <VERSION> in place of those.

**Comment by kaminek** [comment]
> @kaminek: which links? our readme just has in place of those.

hard to link since md is rendered, like this example: https://github.com/UpCloudLtd/upcloud-cli/commit/4d860e53a84180d6eaee313c73f46dba38538c94#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL49-R49

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- New go-api version
+- Improved error messages
+
+### Fixes
+- No coloring of texts if stdout is not a terminal
+- Detaching routers works now with the new `--detach-router` parameter in `network modify`
+
+## [1.0.0] - 2021-04-16
+First non-beta release! Includes all previous changes and fixes.
+
```
Comment (Darep) [inline CHANGELOG.md:28]: Should update the links below, at the end of this file to match 👍

Comment (Darep) [reply CHANGELOG.md:28]: Ahh yes, Amine just commented about the same thing 👍

**Pull request merged**